plaidml/bridge/openvino/src/plaidml_plugin/plaidml_op_clamp.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_op_clamp.hpp"
+#include "plaidml_util.hpp"
+
+#include "plaidml/op/op.h"
+
+void OpClamp::run(const plaidml::edsl::Tensor& I, plaidml::edsl::Tensor& O) {
+  auto clamp = dynamic_cast<ClampLayer*>(layer_.get());
+
+  O = plaidml::edsl::select(I > clamp->max_value, Tensor(clamp->max_value), I);
+  O = plaidml::edsl::select(O < clamp->min_value, Tensor(clamp->min_value), O);
```

Comment:
The repeated use of O _looks_ worrisome to me, but I think that may be me remembering pre-EDSL tile. I think with the EDSL it's fine. (But I'm flagging in case @flaub knows a reason it isn't fine.)